02LessonTwo/in-class-project/index.html

```diff
@@ -5,6 +5,21 @@
   <link rel="stylesheet" href="css/style.css">
```

Comment:
You missed to add something to the title element. That's why I don't see any change on that line